src/cosalib/metal.py

```diff
@@ -0,0 +1,316 @@
+"""
+Base Class for building Metal-type disks.
+"""
+
+# Copyright 2018-2020 Red Hat, Inc
+# Licensed under the new-BSD license (http://www.opensource.org/licenses/bsd-license.php)
+
+import distutils.util as dutil
+import logging as log
+import os
+import os.path
+import sys
+import yaml
+
+cosa_dir = os.path.dirname(os.path.abspath(__file__))
+sys.path.insert(0, f"{cosa_dir}/cosalib")
+sys.path.insert(0, cosa_dir)
+
+from cosalib.build import (
+    _Build,
+    BuildExistsError
+)
+from cosalib.ostree import _BuildOSTree
+from cosalib.cmdlib import (
+    get_basearch,
+    run_verbose,
+)
+
+
+# For padded, disk is actual plus 513M
+SIZE_PADDED = "513M"
+# For estimated sizes, inflate the actual rootfs by this percent
+SIZE_ESITMATED = 35
```

Comment:
```suggestion
SIZE_ESTIMATED = 35
```

Reply:
I OVERESITMATE all the time :stuck_out_tongue: